test/smoke/src/areas/positron/variables/variables-expanded.test.ts

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from '@playwright/test';
+import { Application, PositronPythonFixtures } from '../../../../../automation';
+import { setupAndStartApp } from '../../../test-runner/test-hooks';
+
+describe('Variables - Expanded View #pr #web #win', () => {
```

Comment:
Do you think its a good idea to add this as a #pr case?  Just asking because we usually wait until we are sure a test is solid and free from flakes before adding to the pr suite.

Reply:
i've ran it a bunch and it has always passed, but i can remove if we want to build confidence first... not a big deal.

Reply:
you can leave it if it finishing under 15 mins for PRs